tests/jobs/test_backfill_job.py

```diff
@@ -287,10 +284,7 @@ def test_backfill_conf(self, dag_maker):
         )
         job.run()
 
-        # We ignore the first dag_run created by fixture
-        dr = DagRun.find(
-            dag_id='test_backfill_conf', execution_start_date=DEFAULT_DATE + datetime.timedelta(days=1)
-        )
+        dr = DagRun.find(dag_id='test_backfill_conf')
```

Comment:
```suggestion
        # We ignore the first dag_run created by fixture
        dr = DagRun.find(
            dag_id='test_backfill_conf', execution_start_date=DEFAULT_DATE + datetime.timedelta(days=1)
        )
```
Does this affect your change?